src/component/SettingsForm/index.jsx

```diff
@@ -48,10 +50,29 @@ const SettingsForm = () => {
           </div>
         </div>
       ))}
-      <div className=" d-flex flex-row justify-content-center">
-        <Button variant="outline-dark" type="submit" className="w-25 mt-5">Save</Button>
+
+      <h5 className="mt-4 d-flex flex-row justify-content-center">Start Automaticaly Next period</h5>
```

Comment:
it's the space after the quotation, right?
<div className="d-flex flex-row justify-content-center"> now it's true?

Reply:
Yeah the space after the quotation was wrong, but you already fixed that. I'm talking about a spelling mistake.

Hint: turn on spell check and let it tell you where you made the spelling mistake.

Reply:
thank you for your hint, I turn on the spell check in VS code, there is no mistake at the place you point at.

Reply:
Seems like your VSCode  spell check is not working for you then. Copy paste the code into another spell check engine and detect the spelling error that way.

Reply:
I think the error in lines (51 & 52) because of the red color.
 The mistake is the word (Automaticaly) in line 54 ----> the correct is (Automatically), that's right?

Reply:
That is correct 👍